src/App.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import Homepage from "./pages/Homepage";
+import SignUpPage from "./pages/SignUpPage";
 
 function App() {
   return (
-    <Homepage/>
+    // <Homepage/>
```

Comment:
As the instruction of the sign-up page was to create a non-functional (UI only) page, I've commented out `Homepage` from App.js file. Please let me know if i need to use React Router function. Thanks!